### PR TITLE
Fix an issue locating the Flutter SDK on Windows when not invoked via 'dart' inside a Flutter SDK

### PR DESCRIPTION
**Pull request by DanTup**
If the current Dart VM isn't inside a Flutter SDK, we fall back to trying to find it on `PATH` with `which`. The equiv on Windows is `where.exe`.

### CODE REVIEW

**Comment by DanTup** [comment]
Question: Can I add the autosubmit label before there's a review, and the bot will merge it once reviews - or does it only work on the bot checks? @CoderDake ?

**Comment by CoderDake** [comment]
@DanTup I'm not 100% sure but we are about to find out 😄

**Review by CoderDake — APPROVED** [review]

**Comment by DanTup** [comment]
I guess my real concern was whether it would merge it when everything went green but there still wasn't an approving review (I don't know whether GitHub's rules apply to the bot). So to test, we should add the tag to a review that is good, but without an approval.

I'll test with https://github.com/flutter/devtools/pull/6669 since it'd be handy to know :)

**Comment by CoderDake** [comment]
@DanTup looks like it submitted on it's own after the approval :D

**Pull request merged**
